src/components/pages/document/show/sidebar/coverage/index.tsx

```diff
@@ -0,0 +1,57 @@
+import { Box, IconButton, useDisclosure } from "@chakra-ui/react";
+import BoxHeading from "@components/@core/layout/box-heading";
+import useTranslation from "@hooks/use-translation";
+import EditIcon from "@icons/edit";
+import { axUpdateCoverageGroup } from "@services/document.service";
+import notification, { NotificationType } from "@utils/notification";
+import React, { useState } from "react";
+
+import CoverageEdit from "./coverage-edit";
+import { CoverageShow } from "./coverage-show";
+
+export default function CoveragePanel({
+  icon,
+  title,
+  initialValue,
+  items,
+  type,
+  documentId,
+  endpointType
+}) {
+  const { t } = useTranslation();
+  const { isOpen, onToggle, onClose } = useDisclosure();
+  const [coverageValue, setCoverageValue] = useState(initialValue);
+
+  const onSave = async (newCoverageValue) => {
+    const { success } = await axUpdateCoverageGroup(endpointType, documentId, newCoverageValue);
+    if (success) {
+      setCoverageValue(newCoverageValue);
+      onClose();
```

Comment:
Can you place the onClose function after the if//else condition

Reply:
![image](https://user-images.githubusercontent.com/5774849/107185290-3405a580-6a08-11eb-9179-a5145a5ad643.png)

hi @teamdraftbox

`onClose()` is purposefully inside if because I don't want to close edit mode since data will be discarded if it got failed so it will just show error message so user can try again

Reply:
I have gone through the changes looks good to me,Please merge the branch, thanx